tensorflow/buildspec.yml

```diff
@@ -42,7 +42,7 @@ images:
   BuildTensorflowE3CpuPy39TrainingDockerImage:
```

Comment:
For all the e3 images, please set `enable_test_promotion: false`. This would prevent us from running tests on e3 images on the mainline.

Reply:
is this comment still relavant?

Reply:
No thanks!!